### ISSUE TITLE
API is missing CORS headers

### ISSUE THREAD

**Issue by Durss** [lumiacove/lumiastream-api-demo-js#8]
Hello!

Currently Lumia REST API lacks CORS headers.
It's an issue if we want to use it from a web app.

Browsers require CORS to accept connections from a given domain to allow it to connect to.
Browsers have other restrictions like cross-security stuff which technically would avoid a secured domain to connect with an unsecured one but on most browsers there's an exception on localhost/127.0.0.1 _(except for some Safari and Brave if shield is enabled)_ so if CORS headers were added we could technically connect to Lumia API.

Basically it would be nice if Lumia was sending this in the response headers:
`Access-Control-Allow-Origin: *`

Have a good day :)

**Comment by ahadcove** [comment]
> Access-Control-Allow-Origin: *

Ahh good catch, we'll add it to the next version. Beta may go out today for this for 7.5.3 as well
